components/Python/fileConnectors/s3_file_source/component.json

```diff
@@ -0,0 +1,92 @@
+{
+  "engineType": "Python",
+  "userStandalone": false,
+  "language": "Python",
+  "name": "s3_file_source",
+  "label": "S3 File Source",
+  "description": "Import a bucket/key from S3 to a file",
+  "program": "main.py",
+  "componentClass": "S3FileSource",
+  "group": "Connectors",
+  "useMLOps": true,
+  "inputInfo": [],
+  "outputInfo": [
+    {
+        "description": "File name",
+        "label": "File-Name",
```

Comment:
File-Name  with capital letters and dash look very strange :)

Reply:
replaced it to "filename"